content/en/docs/languages/go/generated-code.md

```diff
@@ -34,68 +35,29 @@ In this context, `MsgA` is the protobuf message sent from the client, and `MsgB`
 ### Server-streaming methods
 
 These methods have the following signature on the generated service interface:
+`Foo(*MsgA, grpc.ServerStreamingServer[*MsgB]) error`
 
-`Foo(*MsgA, <ServiceName>_FooServer) error`
-
-In this context, `MsgA` is the single request from the client, and the `<ServiceName>_FooServer` parameter represents the server-to-client stream
-of `MsgB` messages.
-
-`<ServiceName>_FooServer` has an embedded `grpc.ServerStream` and the following interface:
-
-```go
-type <ServiceName>_FooServer interface {
-	Send(*MsgB) error
-	grpc.ServerStream
-}
-```
-
-The server-side handler can send a stream of protobuf messages to the client through this parameter's `Send` method. End-of-stream for the server-to-client
```

Comment:
Since all this documentation was deleted, let's explicitly call out that instructions on how to use the Stream type can be found at the pkg.go.dev link above.

Reply:
Done